rmf_api_msgs/schemas/task_schedule_remove_request.json

```diff
@@ -0,0 +1,20 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://raw.githubusercontent.com/open-rmf/rmf_api_msgs/main/rmf_api_msgs/schemas/task_schedule_remove_request.json",
+  "title": "Add Task Schedule Request",
+  "description": "Request a task to be executed at a fixed schedule",
+  "type": "object",
+  "properties": {
+    "type": {
+      "description": "Indicate that this is a task schedule remove request",
+      "type": "string",
+      "constant": "task_schedule_remove_request"
+    },
+    "name": {
+      "description": "(Optional) Name for the scheduled task, if a task with this name already exists the schedule will be updated. Will be autogenerated if not provided",
+      "type": "string"
+    }
+  },
+  "required": ["name"]
```

Comment:
The description says it is optional but the schema says it is required. I think the description is wrong, I don't see how we can auto generate a name to remove.

Reply:
Copy paste error, done https://github.com/open-rmf/rmf_api_msgs/pull/25/commits/0fa0d79ce48ecfd21a31472b0004497511ac6594